indico/modules/events/papers/templates/management/assignment.html

```diff
@@ -0,0 +1,95 @@
+                <ul class="dropdown">
+                    <li>
+                        <a href="#" data-select-all="#assignment-list input:checkbox">{% trans %}All{% endtrans %}</a>
+                    </li>
+                    <li>
+                        <a href="#" data-select-none="#assignment-list input:checkbox">{% trans %}None{% endtrans %}</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="group">
+                <button class="i-button icon-settings js-dialog-action js-customize-list highlight"
+                        data-href="#"
+                        data-title="{% trans %}Paper assignment list configuration{% endtrans %}"
+                        data-dialog-classes="list-filter-dialog"
+                        data-update='{"html": "#assignment-list", "filter_statistics": "#filter-statistics"}'
+                        data-ajax-dialog>
+                    {% trans %}Customize list{% endtrans %}
+                </button>
+            </div>
+            <div class="group">
+                <a class="i-button arrow js-dropdown" data-toggle="dropdown">
+                    {%- trans %}Assign{% endtrans -%}
+                </a>
+                <ul class="dropdown">
+                    <li><a href="#">{% trans %}Judge{% endtrans %}</a></li>
+                    <li><a href="#">{% trans %}Content reviewer{% endtrans %}</a></li>
+                    <li><a href="#">{% trans %}Layout reviewer{% endtrans %}</a></li>
+                </ul>
+                <a class="i-button">
+                    {%- trans %}Unassign{% endtrans -%}
+                </a>
+            </div>
+            <div class="group">
+                <a href="#" class="i-button icon-checkmark js-enable-if-checked disabled"
+                   data-href="#">
+                    {%- trans %}Accept{% endtrans -%}
+                </a>
+            </div>
+            <div class="group">
+                <a href="#" class="i-button icon-cross js-enable-if-checked disabled"
+                   data-href="#">
+                    {%- trans %}Reject{% endtrans -%}
+                </a>
+            </div>
+            <div class="group">
+                <a href="#" class="i-button icon-quill js-enable-if-checked disabled"
+                   data-href="#">
+                    {%- trans %}To be corrected{% endtrans -%}
+                </a>
+            </div>
+            <div class="group">
+                <a href="#" class="i-button icon-attachment js-enable-if-checked disabled"
+                   data-href="#">
+                    {%- trans %}Download papers{% endtrans -%}
+                </a>
+            </div>
+        </div>
+        <div class="toolbar">
+            <div class="group" id="filter-statistics">
+                {{ render_displayed_entries_fragment(contribs|length, total_entries) }}
```

Comment:
Where did the denominator go? :smile: 
![image](https://cloud.githubusercontent.com/assets/1579899/21054724/03dd61ce-be2f-11e6-8506-9c5bdeea340e.png)